sphinx/source/docs/user_guide/categorical.rst

```diff
@@ -18,17 +18,17 @@ Bars
 Basic
 ~~~~~
 
-Bokeh make it simple to create basic bar charts using the
-:func:`~bokeh.plotting.Figure.hbar` and
-:func:`~bokeh.plotting.Figure.vbar` glyphs methods. In the example
+Bokeh makes it simple to create basic bar charts using the
+:func:`~bokeh.plotting.figure.Figure.hbar` and
+:func:`~bokeh.plotting.figure.Figure.vbar` glyphs methods. In the example
```

Comment:
perhaps we should figure this out before merging. The physical module is `bokeh.plotting.figure` but the public API is intended to be `bokeh.plotting`

Reply:
It's possible there was a recent Sphinx change that bears on this

Reply:
I did not knowingly change anything in these directives. It seems to me like my fork missed your pull request #10222 for fixing the docs for Sphinx 3.1.1. I'll go through my changes and make sure that my pull request is in sync with the changes made in #10222. My next commit to this pull request will hopefully resolve this. I'm sorry about the hassle!

Reply:
👍